src/sas/qtgui/Plotting/PlotterBase.py

```diff
@@ -17,6 +17,49 @@
 
 DEFAULT_CMAP = mpl.cm.jet
 
+class CustomToolbar(NavigationToolbar):
+    def __init__(self, canvas, parent=None):
+        super().__init__(canvas, parent)
+        self.parent = parent
+        self.addSeparator()
+        self.add_custom_button()
+
+
+
+    def add_custom_button(self):
+        # I have been told that a Button is better
+        # But all NavitaionToolbar are interactions are Actions
+        # This way all can be called with:
+        #   self._actions['xxx']
+        custom_icon = QtGui.QIcon()  # You can load an icon here if you want e.g., QtGui.QIcon("path/to/icon.png")
+        custom_action = QtGui.QAction(custom_icon, "Fitting", self)
```

Comment:
"Fitting" is not descriptive enough, even with the tootlip. Please consider e.g. "Send to fitting" or even better "Freeze" since this is what the operation does.